CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format located [here](https://github.com/sensu-plugin
 
 ## [Unreleased]
 
+## [2.0.3] - 2018-09-14
```

Comment:
In general the versioning and dating are left to the maintainers for several reasons:
- we may disagree on the impact of the change and therefore how its versioned (in this case it would be a minor so it would be `2.1.0`) per our [versioning guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_VERSION.md)
- we date the release rather then when the contribution was submitted, PR review and acceptance has no SLA and is not guaranteed to be merged within the same day
- PRs are not a FIFO queue, as such versioning them prior to acceptance leads to needing to ask people to rebase more often